src/config.cpp

```diff
@@ -836,6 +836,10 @@ namespace config {
     std::vector<std::string> list;
     list_string_f(vars, name, list);
 
+    // The framerate list must be cleared before adding values from the file configuration. 
+    // If the list is not cleared, then the specified parameters do not affect the behavior of the sunshine server. 
+    // That is, if you set only 30 fps in the configuration file, it will not work because by default, during initialization the list includes 10, 30, 60, 90 and 120 fps.
+    input.clear();
```

Comment:
```suggestion
    // The framerate list must be cleared before adding values from the file configuration.
    // If the list is not cleared, then the specified parameters do not affect the behavior of the sunshine server.
    // That is, if you set only 30 fps in the configuration file, it will not work because by default, during initialization the list includes 10, 30, 60, 90 and 120 fps.
    input.clear();
```